### PR TITLE
279 create footer

### PR DESCRIPTION
**Pull request by JoeYlMS**
Create Footer ,
added footer in layout;

added pages :
about_service,
about_us,
faq,

### CODE REVIEW

File: client/src/components/footer/config.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    Icon: EmailSocial,
+    link: 'it.volunteers.ukraine@gmail.com',
+  },
+];
+
+export interface INavigate {
+  id: string;
+  link: string;
+  title: string;
+}
```
Comment (Wolshebnik) [inline client/src/components/footer/config.ts:32]: put it in a file called types

File: client/src/components/footer/config.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    id: '4',
+    link: routes.contacts,
+    title: 'contacts',
+  },
+];
+
+export interface IConfigFooter {
+  social: ISocial[];
+  navigate: INavigate[];
+}
```
Comment (Wolshebnik) [inline client/src/components/footer/config.ts:60]: put it in a file called types

File: client/src/components/footer/config.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import { EmailSocial, FacebookSocial, InstagramSocial } from '@/assets/icons';
+import { routes } from '@/constants';
```
Comment (Wolshebnik) [inline client/src/components/footer/config.ts:2]: sort

File: client/src/components/footer/config.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+export interface ISocial {
+  id: string;
+  Icon: React.ComponentType<React.SVGProps<SVGSVGElement>>;
+  link: string;
+}
```
Comment (Wolshebnik) [inline client/src/components/footer/config.ts:8]: put it in a file called types

File: client/src/components/footer/config.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    link: routes.aboutService,
+    title: 'aboutService',
+  },
+  {
+    id: '3',
+    link: routes.faq,
+    title: 'faq',
+  },
+  {
+    id: '4',
```
Comment (Wolshebnik) [inline client/src/components/footer/config.ts:51]: why need id?

File: client/src/components/footer/index.tsx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+import Link from 'next/link';
+import { useRouter } from 'next/navigation';
+import { useTranslations } from 'next-intl';
+
+import { routes } from '@/constants';
+
+import { Logo } from '../logo';
+import { Button } from '../button';
+
```
Comment (Wolshebnik) [inline client/src/components/footer/index.tsx:11]: delete space and sort

File: client/src/components/footer/index.tsx [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+  return (
+    <footer className="flex w-full flex-col gap-14 bg-boardAside px-4 py-8 tablet:gap-12 tablet:px-8 tablet:pt-14 laptop:px-9 desktop:gap-10 desktop:px-[136px] desktopXl:px-[92px]">
+      <div className="flex flex-col justify-between gap-10 tablet:flex-row tablet:gap-0">
+        <div className="flex flex-col gap-6">
+          <Logo className="desktop:pl-1" />
+
+          <div className="flex gap-x-2">
+            {social.map((item, idx) => (
+              <Messengers key={item.id + '_' + idx} {...item} />
```
Comment (Wolshebnik) [inline client/src/components/footer/index.tsx:31]: just use
key={`social_ ${idx}}

File: client/src/components/footer/config.ts [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+export interface INavigate {
+  id: string;
+  link: string;
+  title: string;
+}
+
+const navigate: INavigate[] = [
+  {
+    id: '1',
```
Comment (Wolshebnik) [inline client/src/components/footer/config.ts:36]: Why need Id?

File: client/src/components/footer/index.tsx [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+            ))}
+          </div>
+        </div>
+
+        <div className="flex flex-col gap-4 tablet:pr-32 laptop:pr-0 lg:mr-0 lg:flex-row lg:gap-8 desktop:flex-row desktop:gap-16 desktopXl:gap-[108px]">
+          {navigate.map((item, idx) => (
+            <Navigate key={item.id + '_' + idx} {...item} />
```
Comment (Wolshebnik) [inline client/src/components/footer/index.tsx:38]: just use
key={`navigate_ ${idx}}

File: client/src/components/footer/index.tsx [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+            text={auth('registration')}
+            onClick={() => router.push(routes.registration)}
+            className="h-fit px-2 uppercase"
+          />
+        </div>
+      </div>
+      {
+        // TODO add a route to the privacy policy link
+      }
+      <Link href={'#'} className="block text-center font-scada text-xs text-white">
```
Comment (Wolshebnik) [inline client/src/components/footer/index.tsx:61]: curly brackets no need

**Review by VladDot — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**